internal/resources/standalone/bindata.go

```diff
@@ -24,15 +24,15 @@ import (
 func bindataRead(data []byte, name string) ([]byte, error) {
 	gz, err := gzip.NewReader(bytes.NewBuffer(data))
 	if err != nil {
-		return nil, fmt.Errorf("read %q: %v", name, err)
+		return nil, fmt.Errorf("Read %q: %v", name, err)
```

Comment:
Ick. Looks like the new go-bindata generates code that `staticcheck` does not like :/

I think we'll need to add an exception somewhere. I think we can maybe update the Makefile to try to exclude generated files when it runs the checker.